requirements/pip.txt

```diff
@@ -17,7 +17,7 @@ Sphinx==1.8.5  # pyup: <2.0.0
 # Filtering for the REST API
 django-filter==2.2.0
 
-drf-flex-fields==0.5.0
+drf-flex-fields==0.6.1
```

Comment:
From the changelog

> Moves declaration of expandable_fields to Meta class on serialzer for consistency with DRF (will continue to support declaration as class property)

We should update those fields, I'm creating a new issue to track that